mempool/src/pool/tests.rs

```diff
@@ -1879,3 +1879,94 @@ async fn no_empty_bags_in_indices(#[case] seed: Seed) -> anyhow::Result<()> {
     mempool.store.assert_valid();
     Ok(())
 }
+
+struct TestTxAccumulator {
```

Comment:
You can call this DefaultTxAccumulator, and make it available for block production. This way you don't have to do it twice, but a test that needs a custom accumulator can write one.

Reply:
Done.